runtime/bcutil/cfreader.c

```diff
@@ -2397,6 +2388,13 @@ j9bcutil_readClassFileBytes(J9PortLibrary *portLib,
 
 	NEXT_U16(classfile->minorVersion, index);
 	NEXT_U16(classfile->majorVersion, index);
+
+	/* Make sure that following verification uses the class file version number. VM version is
```

Comment:
Since this code was moved, it should change to say "the structure and static verification below" instead of "following verification"

Reply:
Just updated the comment.